tests/unit/components/__snapshots__/HdLazyImage.spec.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`HdLazyImage renders with the image as background 1`] = `<picture> <img class="lazy-image" src=""></picture>`;
```

Comment:
Shouldn't this guy render a `<div>` then?

Reply:
Great cache @viniciuskneves . It should, both snapshots are rendered empty (no data.)